modules/Emails/Emails.php

```diff
@@ -298,6 +298,9 @@ public static function sendEMail(
 			$mail->AddReplyTo($HELPDESK_SUPPORT_EMAIL_REPLY_ID);
 		}
 		$mail_status = MailSend($mail);
+
+		cbEventHandler::do_action('corebos.email.aftersend', $mail);
```

Comment:
add the $mail_status to the parameters that the event receives and I will accept. LGTM

Reply:
Can we do that by just adding a parameter to the `do_action` call or do I need to wrap it in an array?

Reply:
has to be wrapped in an array

Reply:
Done. The receiving function should unwrap that array right? That doesn't happen automagically?

Reply:
Yes, correct